exhibitions/app/controllers.js

```diff
@@ -81,3 +83,56 @@ async function getExhibitionAndRelatedContent(id: string, previewReq: ?Request):
     relatedArticles: relatedArticles
   };
 }
+
+export async function renderExhibitionsList(ctx, next) {
+  const page = Number(ctx.request.query.page);
+  const allExhibitions = await getExhibitions(page);
+  const currentPage = allExhibitions && allExhibitions.page;
+  const pageSize = allExhibitions && allExhibitions.results_per_page;
+  const totalResults = allExhibitions && allExhibitions.total_results_size;
+  const totalPages = allExhibitions && allExhibitions.total_pages;
```

Comment:
We could abstract this out into a factory as we'll use it in a few places?
https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/services/prismic.js#L66

Reply:
And wonder how this sits in the conversation here:
https://github.com/wellcometrust/wellcomecollection.org/pull/1890#discussion_r154339998